docs/api/browser-window.md

```diff
@@ -1188,3 +1191,17 @@ Returns `BrowserWindow[]` - All child windows.
 
 [window-levels]: https://developer.apple.com/reference/appkit/nswindow/1664726-window_levels
```

Comment:
Usually we leave these footer links at the very bottom of the docs so the new docs can go above here.